go.mod

```diff
@@ -9,14 +9,14 @@ require (
 	github.com/mroth/weightedrand v0.4.1
 	github.com/pmezard/go-difflib v1.0.0
 	github.com/sergi/go-diff v1.2.0
+	github.com/spaolacci/murmur3 v1.1.0
```

Comment:
Why was this moved to be a direct dependency?

Reply:
I'm not really sure @carreter. It's used only in the `mash` package similar to how blake3 is only used in the `seqhash` package.

Reply:
Yeah had to double check but murmur3 should have been a direct dependency to begin with @carreter. Good to merge?